### PR TITLE
Wardrobe feature and fix answer reshuffle bug

### PR DESCRIPTION
**Pull request by SimpleLuke**
- Wardrobe feature: user can change outfit through wardrobe
- By default, there are three outfits in the inventory
- Disable answer button when user clicked once per question
- Ryan fixed the answer reshuffle bug

image

### CODE REVIEW

File: frontend/src/components/wardrobe/Wardrobe.component.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        ベッドルームに戻る - Back To Bedroom
+      </button>
+
+      <div className="flex px-10">
+        <div className="flex justify-center items-center border-black border-2 rounded-lg w-1/3 h-[calc(100vh-100px)] mx-auto max-w-2xl px-4 py-16 sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8">
+          <CharacterComponent data={character.currentOutfit} />
+        </div>
+        <div className="w-2/3 mx-auto max-w-2xl px-4 py-16 sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8">
+          <h2 className="text-xl font-bold text-gray-900">
+            Which one you want to change?
```
Comment (edwardfgt) [inline frontend/src/components/wardrobe/Wardrobe.component.js:101]: English correction - "Which one do you want to change?"

Although we could change it to something else too.

**Review by edwardfgt — APPROVED** [review]
Great job!

File: backend/src/routes/users/users.controller.js [hunk 2]
```diff
@@ -9,13 +9,6 @@ const UsersController = {
     } catch (err) {
       return res.status(400).json({ message: "Bad request" });
     }
-    // await user.save((err) => {
```
Comment (edwardfgt) [inline backend/src/routes/users/users.controller.js:12]: Commented out code can be removed.

File: frontend/src/components/wardrobe/Wardrobe.component.js (hunk 1 above)
Comment (forreya) [reply frontend/src/components/wardrobe/Wardrobe.component.js:101]: Maybe "Pick your outfit" would be better?

**Review by forreya — APPROVED** [review]

**Pull request merged**